pivotTable.go

```diff
@@ -459,12 +459,16 @@ func inPivotTableField(a []PivotTableField, x string) int {
 // addPivotColFields create pivot column fields by given pivot table
 // definition and option.
 func (f *File) addPivotColFields(pt *xlsxPivotTableDefinition, opt *PivotTableOption) error {
+	pt.ColFields = &xlsxColFields{}
 	if len(opt.Columns) == 0 {
+		//in order to create pivot table in case there is no input from Columns 
+		pt.ColFields.Count = 1
+		pt.ColFields.Field = append(pt.ColFields.Field, &xlsxField{
+			X: -2,
```

Comment:
This modification will cause Excel application crash when there are multiple pivot tables in a worksheet. I think we need to resolve this issue in other way.